src/pages/Fox/components/FoxGovernance.tsx

```diff
@@ -0,0 +1,205 @@
+import type { FlexProps } from '@chakra-ui/react'
+import {
+  Box,
+  Card,
+  CardBody,
+  Divider,
+  Flex,
+  Heading,
+  Link,
+  Skeleton,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  Text as CText,
+} from '@chakra-ui/react'
+import { foxAssetId } from '@shapeshiftoss/caip'
+import { useCallback, useEffect, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useDispatch } from 'react-redux'
+import { Amount } from 'components/Amount/Amount'
+import { Text } from 'components/Text'
+import { useFeatureFlag } from 'hooks/useFeatureFlag/useFeatureFlag'
+import {
+  selectActiveProposals,
+  selectClosedProposals,
+  selectIsSnapshotApiQueriesPending,
+  selectVotingPower,
+} from 'state/apis/snapshot/selectors'
+import { snapshotApi } from 'state/apis/snapshot/snapshot'
+import { selectAssetById, selectWalletAccountIds } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { FoxGovernanceProposal } from './FoxGovernanceProposal'
+
+const containerPaddingX = { base: 4, xl: 0 }
+const headerTitleMb = { base: 4, md: 0 }
+const headerTitleMaxWidth = { base: '100%', md: '50%' }
+const headerSx: FlexProps['sx'] = {
+  alignItems: { base: 'flex-start', md: 'center' },
+  justifyContent: 'space-between',
+  mb: 8,
+  flexDir: {
+    base: 'column',
+    md: 'row',
+  },
+}
+
+const flexPropsMd1 = { base: '1 0 auto', md: 1 }
+const flexPropsMdAuto = { base: 1, md: 'auto' }
+const widthBaseFull = { base: 'full' }
+const widthMdAuto = { base: 'full', md: 'auto' }
+const tabListPaddingLeft = { base: 6, md: 0 }
+
+export const FoxGovernance = () => {
+  const translate = useTranslate()
+  const isFoxGovernanceEnabled = useFeatureFlag('FoxPageGovernance')
+  const dispatch = useDispatch()
+  const accountIds = useAppSelector(selectWalletAccountIds)
+
+  const foxEthAsset = useAppSelector(state => selectAssetById(state, foxAssetId))
+
+  const votingPower = useAppSelector(state => selectVotingPower(state, { feeModel: 'SWAPPER' }))
+  const isVotingPowerQueriesPending = useAppSelector(selectIsSnapshotApiQueriesPending)
+  const activeProposals = useAppSelector(selectActiveProposals)
+  const closedProposals = useAppSelector(selectClosedProposals)
+
+  const firstFifthClosedProposals = useMemo(() => closedProposals.slice(0, 5), [closedProposals])
+
+  useEffect(() => {
+    dispatch(snapshotApi.endpoints.getProposals.initiate())
+  }, [dispatch])
+
+  useEffect(() => {
+    dispatch(
+      snapshotApi.endpoints.getVotingPower.initiate({ model: 'SWAPPER' }, { forceRefetch: true }),
+    )
+  }, [dispatch, accountIds])
+
+  console.log({ votingPower, accountIds })
```

Comment:
ser

Reply:
Fixed in [2b38bd6](https://github.com/shapeshift/web/pull/7952/commits/2b38bd691a2b53a6c5fc056cd4fe189082ca3806)